com.xargon.retrodeck.yml

```diff
@@ -0,0 +1,643 @@
+      - libdecor/libdecor-0.1.0.json
+  - name: retroarch-filers-video
+    subdir: gfx/video_filters
+    make-install-args:
+      - PREFIX=${FLATPAK_DEST}
+    sources:
+      - type: git
+        url: https://github.com/libretro/RetroArch.git
+        commit: de4e56ecf28a09057734f8624af6d62f3088734f
+  - name: retroarch-filers-audio
+    subdir: libretro-common/audio/dsp_filters
+    make-install-args:
+      - PREFIX=${FLATPAK_DEST}
+    sources:
+      - type: git
+        url: https://github.com/libretro/RetroArch.git
+        commit: de4e56ecf28a09057734f8624af6d62f3088734f
+  - name: retroarch-assets
+    make-install-args:
+      - PREFIX=${FLATPAK_DEST}
+    sources:
+      - type: git
+        url: https://github.com/libretro/retroarch-assets.git
+        commit: f45dd1351e6bfb9f01e12bee8d358ad22a2bb8f9
+  - name: libretro-database
+    make-install-args:
+      - PREFIX=${FLATPAK_DEST}
+    sources:
+      - type: git
+        url: https://github.com/libretro/libretro-database.git
+        commit: 1754130f72d8c00a512797bf85442576258279e8
+  - name: libretro-core-info
+    make-install-args:
+      - PREFIX=${FLATPAK_DEST}
+    sources:
+      - type: git
+        url: https://github.com/libretro/libretro-core-info.git
+        commit: 1b2472eca14a5a187ff298ace1b614fd3d47bc7c
+  - name: retroarch-joypad-autoconfig
+    make-install-args:
+      - PREFIX=${FLATPAK_DEST}
+    sources:
+      - type: git
+        url: https://github.com/libretro/retroarch-joypad-autoconfig.git
+        commit: 0eb3ad2666e6bdc2de2cde4c5d4af29b8c7f2c67
+  - name: common-shaders
+    make-install-args:
+      - PREFIX=${FLATPAK_DEST}
+    sources:
+      - type: git
+        url: https://github.com/libretro/common-shaders.git
+        commit: 55e401834b732e62c34411321c4ffd82524345d4
+  - name: slang-shaders
+    make-install-args:
+      - PREFIX=${FLATPAK_DEST}
+    sources:
+      - type: git
+        url: https://github.com/libretro/slang-shaders.git
+        commit: 77558d5f2ac807b27201fb888cbf2caee8e289fa
+  - name: glsl-shaders
+    make-install-args:
+      - PREFIX=${FLATPAK_DEST}
+    sources:
+      - type: git
+        url: https://github.com/libretro/glsl-shaders.git
+        commit: 20801b38aa2c89a2beb9f57ffbbbcea7256e2523
+  - name: common-overlays
+    make-install-args:
+      - PREFIX=${FLATPAK_DEST}
+    sources:
+      - type: git
+        url: https://github.com/libretro/common-overlays.git
+        commit: db9744f4e58a740f0f10b04b62af347cd6f01928
+
+  # RetroArch - END
+
+  # Not part of the offical RetroArch manifest
+  - name: retroarch-cores
+    buildsystem: simple
+    build-commands:
+        - mkdir -p /app/share/libretro/cores/
+        - mv ./RetroArch-Linux-x86_64.AppImage.home/.config/retroarch/cores/* /app/share/libretro/cores/
+    cleanup:
+        - RetroArch_cores.7z
+    sources:
+      - type: archive
+        url: https://buildbot.libretro.com/stable/1.10.2/linux/x86_64/RetroArch_cores.7z
```

Comment:
you just made the whole package x86_64 only.

Reply:
Yes as Steam Deck is x86_64

Reply:
Flathub != Steamdeck. We have aarch64 users too.

Reply:
https://github.com/flathub/flathub/wiki/App-Maintenance#limiting-the-set-of-architectures-to-build-on

Reply:
Done